installer/bundle/bundle_skel.sh

```diff
@@ -96,64 +99,87 @@ cleanup_and_exit()
     fi
 }
 
-check_version_installable() {
+first_version_lower_than_second() {
     # POSIX Semantic Version <= Test
-    # Exit code 0 is true (i.e. installable).
-    # Exit code non-zero means existing version is >= version to install.
+    # Exit code 0 means VersionA < VersionB
+    # Exit code non-zero means VersionA >= VersionB
     #
     # Parameter:
-    #   Installed: "x.y.z.b" (like "4.2.2.135"), for major.minor.patch.build versions
-    #   Available: "x.y.z.b" (like "4.2.2.135"), for major.minor.patch.build versions
+    #   VersionA: "x.y.z.b" (like "4.2.2.135"), for major.minor.patch.build versions
+    #   VersionB: "x.y.z.b" (like "4.2.2.135"), for major.minor.patch.build versions
 
     if [ $# -ne 2 ]; then
-        echo "INTERNAL ERROR: Incorrect number of parameters passed to check_version_installable" >&2
+        echo "INTERNAL ERROR: Incorrect number of parameters passed to first_version_lower_than_second" >&2
         cleanup_and_exit 1
     fi
 
     # Current version installed
-    local INS_MAJOR=`echo $1 | cut -d. -f1`
-    local INS_MINOR=`echo $1 | cut -d. -f2`
-    local INS_PATCH=`echo $1 | cut -d. -f3`
-    local INS_BUILD=`echo $1 | cut -d. -f4`
+    local A_MAJOR=`echo $1 | cut -d. -f1`
+    local A_MINOR=`echo $1 | cut -d. -f2`
+    local A_PATCH=`echo $1 | cut -d. -f3`
+    local A_BUILD=`echo $1 | cut -d. -f4`
 
     # Available version number
-    local AVA_MAJOR=`echo $2 | cut -d. -f1`
-    local AVA_MINOR=`echo $2 | cut -d. -f2`
-    local AVA_PATCH=`echo $2 | cut -d. -f3`
-    local AVA_BUILD=`echo $2 | cut -d. -f4`
+    local B_MAJOR=`echo $2 | cut -d. -f1`
+    local B_MINOR=`echo $2 | cut -d. -f2`
+    local B_PATCH=`echo $2 | cut -d. -f3`
+    local B_BUILD=`echo $2 | cut -d. -f4`
 
     # Check bounds on MAJOR
-    if [ $INS_MAJOR -lt $AVA_MAJOR ]; then
+    if [ $A_MAJOR -lt $B_MAJOR ]; then
         return 0
-    elif [ $INS_MAJOR -gt $AVA_MAJOR ]; then
+    elif [ $A_MAJOR -gt $B_MAJOR ]; then
         return 1
     fi
 
     # MAJOR matched, so check bounds on MINOR
-    if [ $INS_MINOR -lt $AVA_MINOR ]; then
+    if [ $A_MINOR -lt $B_MINOR ]; then
         return 0
-    elif [ $INS_MINOR -gt $AVA_MINOR ]; then
+    elif [ $A_MINOR -gt $B_MINOR ]; then
         return 1
     fi
 
     # MINOR matched, so check bounds on PATCH
-    if [ $INS_PATCH -lt $AVA_PATCH ]; then
+    if [ $A_PATCH -lt $B_PATCH ]; then
         return 0
-    elif [ $INS_PATCH -gt $AVA_PATCH ]; then
+    elif [ $A_PATCH -gt $B_PATCH ]; then
         return 1
     fi
 
     # PATCH matched, so check bounds on BUILD
-    if [ $INS_BUILD -lt $AVA_BUILD ]; then
+    if [ $A_BUILD -lt $B_BUILD ]; then
         return 0
-    elif [ $INS_BUILD -gt $AVA_BUILD ]; then
+    elif [ $A_BUILD -gt $B_BUILD ]; then
         return 1
     fi
 
-    # Version available is idential to installed version, so don't install
+    # VersionA is idential to VersionB
     return 1
 }
 
+check_version_depending_on_force() {
```

Comment:
Will do. I had created this name before I renamed the original `check_version_installable` to `first_version_lower_than_second`